### ISSUE TITLE
2.3.8 Error: Cannot add a bridged Accessory with the same UUID

### ISSUE THREAD

**Issue by dotnetdever** [cellcortex/homebridge-yeelighter#61]
Hi,
i've updated to module and get this error while starting:

14.12.2020, 15:34:44 [Yeelighter] Loading accessory from cache: 0x000000000653a875 (ceiling10)
14.12.2020, 15:34:44 [Yeelighter] Loading accessory from cache: 0x000000000653a875 (ceiling10)
14.12.2020, 15:34:44 [Yeelighter] Loading accessory from cache: 0x000000000653a875 (ceiling10)
14.12.2020, 15:34:44 Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: 4a276c2b-7753-4bff-b652-9e3b8171XXXX
    at Bridge.Accessory._this.addBridgedAccessory (/usr/local/lib/node_modules/@hoobs/hoobs/node_modules/hap-nodejs/dist/lib/Accessory.js:288:27)
    at /usr/local/lib/node_modules/@hoobs/hoobs/bridge/server.js:250:25
    at Array.filter (<anonymous>)
    at Server.restoreCachedPlatformAccessories (/usr/local/lib/node_modules/@hoobs/hoobs/bridge/server.js:211:81)
    at Server.start (/usr/local/lib/node_modules/@hoobs/hoobs/bridge/server.js:159:14)
    at /usr/local/lib/node_modules/@hoobs/hoobs/bridge/cli.js:94:16
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

Do you have any idea? 

Thanks for your response

**Comment by cellcortex** [comment]
Looks like you have 3 times the same accessory in your homebridge cache. Can you try if deleting the cachedAccessories in your Homebridge fixes things?

**Comment by pixxer** [comment]
Same error here. Also setting up a complete new system and install yeelighter creates the issue...

14.12.2020, 18:42:37 [Yeelighter] Loading accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 18:42:37 [Yeelighter] Loading accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 18:42:37 [Yeelighter] Loading accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 18:42:37 Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: f40d5071-893c-46ab-ac95-fcb422b6c831
    at Bridge.Accessory._this.addBridgedAccessory (/usr/src/hoobs/node_modules/hap-nodejs/dist/lib/Accessory.js:288:27)
    at /usr/src/hoobs/bridge/server.js:250:25
    at Array.filter (<anonymous>)
    at Server.restoreCachedPlatformAccessories (/usr/src/hoobs/bridge/server.js:211:81)
    at Server.start (/usr/src/hoobs/bridge/server.js:159:14)
    at /usr/src/hoobs/bridge/cli.js:94:16
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

**Comment by dotnetdever** [comment]
> Looks like you have 3 times the same accessory in your homebridge cache. Can you try if deleting the cachedAccessories in your Homebridge fixes things?
> […](#)

i will try this later this evening. actually my homebrige is not reachable after de-installing the module  ;)

**Comment by pixxer** [comment]
Hey @cellcortex 
Thx for the Update. After installing v2.3.9 i see "Ingnoring duplicate accessory from cache". But there is still an error and the system does not work. 

14.12.2020, 22:13:57 [Yeelighter] Loading accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 22:13:57 [Yeelighter] Loading accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 22:13:57 [Yeelighter] Ingnoring duplicate accessory from cache: 0x00000000124baae8 (ceiling10)
14.12.2020, 22:13:57 Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: f40d5071-893c-46ab-ac95-fcb422b6c831
    at Bridge.Accessory._this.addBridgedAccessory (/usr/src/hoobs/node_modules/hap-nodejs/dist/lib/Accessory.js:288:27)
    at /usr/src/hoobs/bridge/server.js:250:25
    at Array.filter (<anonymous>)
    at Server.restoreCachedPlatformAccessories (/usr/src/hoobs/bridge/server.js:211:81)
    at Server.start (/usr/src/hoobs/bridge/server.js:159:14)
    at /usr/src/hoobs/bridge/cli.js:94:16
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
14.12.2020, 22:13:57 Got SIGTERM, shutting down Bridge...

**Comment by dotnetdever** [comment]
After de-installing an re-installing (+ cleared cache) my log shows the following.

14.12.2020, 22:14:18 [Yeelighter] New ceiling10 [0x000000000653a875] found at yeelight://192.168.178.29:55443
14.12.2020, 22:14:18 [Yeelighter] Accessory created with UUID 3d076c4a-9d57-473d-9992-96c1b5bfXXXX
14.12.2020, 22:14:18 [Yeelighter] Separate Ambient Accessory created with UUID 4a276c2b-7753-4bff-b652-9e3b8171c388
14.12.2020, 22:14:18 [Yeelighter] [0x000000000653a875] no subtype []
14.12.2020, 22:14:18 HAP Warning: Characteristic 000000E3-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 00000043-0000-1000-8000-0026BB765291. Adding anyway.
14.12.2020, 22:14:18 [Yeelighter] [0x000000000653a875#background] registering subtype 0x000000000653a875#background []
14.12.2020, 22:14:18 HAP Warning: Characteristic 000000E3-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 00000043-0000-1000-8000-0026BB765291. Adding anyway.
14.12.2020, 22:14:18 [Yeelighter] [0x000000000653a875] installed as Color temperature light with ambience light []
14.12.2020, 22:14:18 [Yeelighter] New ceiling10 [0x000000000653a875] found at yeelight://192.168.178.29:55443
14.12.2020, 22:14:18 [Yeelighter] Accessory created with UUID 3d076c4a-9d57-473d-9992-96c1b5bff586
14.12.2020, 22:14:18 [Yeelighter] Separate Ambient Accessory created with UUID 4a276c2b-7753-4bff-b652-9e3b8171c388
14.12.2020, 22:14:18 Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: 3d076c4a-9d57-473d-9992-96c1b5bff586
    at Bridge.Accessory._this.addBridgedAccessory (/usr/local/lib/node_modules/@hoobs/hoobs/node_modules/hap-nodejs/dist/lib/Accessory.js:288:27) [...]
14.12.2020, 22:14:18 Got SIGTERM, shutting down Bridge...

the issue is still open. > After downgrading to 2.3.6 everything is fine again

**Comment by PorridgePi** [comment]
Same thing happening to me. Just tried out homebridge and installed this plugin. 
It simply repeatedly error out.
```
[15/12/2020, 13:35:21] [Yeelighter] Initializing Yeelighter platform...
[15/12/2020, 13:35:21] [Yeelighter] New ceiling1 [0x0000000007f0799b] found at yeelight://192.168.2.63:55443
[15/12/2020, 13:35:21] [Yeelighter] Accessory created with UUID 215dcf2c-ac8c-4f47-b948-1fc9292d1856
[15/12/2020, 13:35:21] [Yeelighter] Separate Ambient Accessory created with UUID 9ecf8762-fe64-487d-822a-f091579dc666
[15/12/2020, 13:35:21] [Yeelighter] [0x0000000007f0799b] no subtype []
HAP Warning: Characteristic 000000E3-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 00000043-0000-1000-8000-0026BB765291. Adding anyway.
[15/12/2020, 13:35:21] [Yeelighter] [0x0000000007f0799b] installed as Color temperature light []
[15/12/2020, 13:35:21] [Yeelighter] New ceiling1 [0x0000000007f0799b] found at yeelight://192.168.2.63:55443
[15/12/2020, 13:35:21] [Yeelighter] Accessory created with UUID 215dcf2c-ac8c-4f47-b948-1fc9292d1856
[15/12/2020, 13:35:21] [Yeelighter] Separate Ambient Accessory created with UUID 9ecf8762-fe64-487d-822a-f091579dc666
[15/12/2020, 13:35:22] Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: 215dcf2c-ac8c-4f47-b948-1fc9292d1856
    at Bridge.Accessory._this.addBridgedAccessory (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:416:15)
    at Bridge.Accessory._this.addBridgedAccessories (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:450:12)
    at Server.handleRegisterPlatformAccessories (/usr/lib/node_modules/homebridge/src/server.ts:528:17)
    at HomebridgeAPI.emit (events.js:315:20)
    at HomebridgeAPI.registerPlatformAccessories (/usr/lib/node_modules/homebridge/src/api.ts:316:10)
    at Discovery.YeelighterPlatform.onDeviceDiscovery (/usr/lib/node_modules/homebridge-yeelighter/src/platform.ts:180:16)
    at Discovery.emit (events.js:315:20)
    at Socket.Discovery.onMessage (/usr/lib/node_modules/homebridge-yeelighter/src/discovery.ts:69:12)
    at Socket.emit (events.js:315:20)
    at UDP.onMessage (dgram.js:919:8)
[15/12/2020, 13:35:22] [Yeelighter] [0x0000000007f0799b] ceiling1 Connected []
[15/12/2020, 13:35:22] Got SIGTERM, shutting down Homebridge...
[15/12/2020, 13:35:22] [Yeelighter] [0x0000000007f0799b] temperature light updated {"power":true,"color_mode":2,"bright":30,"hue":0,"sat":0,"ct":4000,"bg_power":false,"bg_bright":0,"bg_hue":0,"bg_sat":0,"bg_ct":0,"bg_lmode":0,"nl_br":0,"active_mode":0,"name":""} []
[15/12/2020, 13:35:27] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
```
Downgrading to v2.3.7 fixes the issue.

**Comment by PorridgePi** [comment]
Seems to be fixed in v2.3.11.
Should the issue be closed?
